ui/apps/platform/src/Containers/Clusters/ClusterDeployment.tsx

```diff
@@ -54,18 +54,21 @@ function ClusterDeployment({
                         direction={{ default: 'column' }}
                         spaceItems={{ default: 'spaceItemsMd' }}
                     >
-                        <Title headingLevel="h3">1. Download files</Title>
+                        <Title headingLevel="h3">1. Configure possibility of future upgrades</Title>
                         <Text>
-                            Download the required configuration files, keys, and scripts. Enabling
-                            automatic upgrades creates a powerful service account in your secured
-                            cluster that will be used to perform the upgrades.
+                            Configuring clusters for future upgrades creates a powerful service
+                            account in your secured cluster that will be used to perform the
+                            upgrades. This is a prerequisite for automated or on-click upgrades of
+                            legacy-installed Secured Clusters to work.
                         </Text>
                         <Switch
-                            label="Automatic upgrades ON: Secured Clusters are upgraded automatically to match Central's version."
-                            labelOff="Automatic upgrades OFF: Secured clusters are not upgraded automatically."
+                            label="Configured for upgrades: Secured Clusters can be upgraded to match Central's version."
+                            labelOff="Not configured for upgrades: Attempts to upgrade Secured Clusters will fail."
```

Comment:
I think some kind of warning that this cannot (trivially) be enabled later on here would be good.

Reply:
Actually it is trivial, but it would require doing few powerful `kubectl apply` commands on the affected cluster. 

I would not start the thread of doing it manually later, cause this could imply that we would need to describe that manual upgrade scenario in the docs. It may be easier for the customer to remove that cluster and add it again with the toggle turned on. 
Let's maybe come back to this issue when first such customer-case/ticket appears. Would that be okay?

Reply:
Thanks!